packages/library/src/lib/index.ts

```diff
@@ -74,13 +78,13 @@ export const sendDatastarEvent = ((
   )
 }) satisfies SendDatastarEvent
 
-// Timeeout allows inspector to attach to all elements before sending the event
-if (!winAny.ds) {
+export function runForInspector(fn: () => Datastar) {
+  // Timeeout allows inspector to attach to all elements before sending the event
```

Comment:
looks like a typo

Reply:
That typo is pre-existing, I just moved it. Can fix it though.